release-notes/kvm/v9.0.1.md

```diff
@@ -0,0 +1,110 @@
+
+- Deploy as a unique app in app collection in control plane clusters.
+- Always set chart CR annotations so update state calculation is accurate.
+- Only update failed Helm releases if the chart values or version has changed.
+- Fix update state calculation and status resource for long running deployments.
+- Handle 503 responses when GitHub Pages is unavailable.
+- Make HTTP client timeout configurable for pulling chart tarballs in AWS China.
+- Switch from dep to go modules
+- Removed usage of legacy chartconfig CRs in Tiller metrics.
+- Added chartmigration resource for migrating from chartconfig to chart CRs.
+- Updated release resource.
+  - Do not wait when installing or updating long running Helm releases.
+  - Use version field from chart CR to reduce number of HTTP requests to pull chart tarballs.
+  - Wait for deleted Helm release before removing finalizer.
+- Updated status resource.
+  - Improve reason field in CR status when installing a chart fails.
+- Removed legacy chartconfig controller.
+- Allowed usage of custom kubernetes domain.
+- Handled timeouts pulling chart tarballs.
+- Removed CPU limits.
+- Updated for Kubernetes 1.16 compatibility.
+- Cancelled the resource reconciliation in case of tarball errors.
+
+### cluster-operator [v0.23.8](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.8)
+
+- Stop setting IC replicas count.
+- Fix cluster deletion by gracefully handling Tenant Cluster API errors.
+- Initial cluster profile detection support.
+
+### coredns v1.6.5 ([Giant Swarm app v1.1.8](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v118-2020-03-20))
+
+- Updated coredns to upstream version 1.6.5.
+- Removed CPU limits.
+- Updated for Kubernetes 1.16 compatibility.
+- Make `autopath` plugin configurable, optional and disabled by default.
+- Allow custom forward configuration destination and options.
+- Add Pod Disruption Budget.
+- Use cluster.kubernetes.clusterDomain instead of cluster.kubernetes.domain for custom DNS suffix.
+
+### kube-state-metrics v1.9.2 ([Giant Swarm app v1.0.4](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v104))
+
+- Removed CPU and memory limits.
+- Remove addon resizer container
+- Adjust RBAC configuration for k8s 1.16 compatibility
+
+### metrics-server v0.3.3 ([Giant Swarm app v1.0.0](https://github.com/giantswarm/metrics-server-app/blob/master/CHANGELOG.md#v100-2020-01-03))
+
+- Updated for Kubernetes 1.16 compatibility.
+
+### nginx-ingress-controller v0.30.0 ([Giant Swarm app v1.6.9](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v169-2020-04-22))
+
+- Restrict PodSecurityPolicy volumes to only those required (removes wildcard).
+- Tune net.ipv4.ip_local_port_range to 1024 65535 as a safe sysctl.
+- Tune net.core.somaxconn to 32768 via an initContainer with privilege escalation.
+- Use 4 worker processes by default.
+- Use upstream default of max-worker-connections of 16384.
```

Comment:
```suggestion
```
This was prior state, got changed in between and now restored again with latest nginx app release, so for customer upgrading from kvm 9.0.0 there was no actual change.